### PR TITLE
Correct path ARDUINO_DIR for Macs

### PR DESCRIPTION
**Pull request by thomaskilian**
Hi, as posted on your blog I send you a correction for the wrong Mac path.
Cheers

### CODE REVIEW

**Comment by sej7278** [comment]
its not wrong, its just that you're using 1.6 not 1.0

**Comment by thomaskilian** [comment]
Hmm, Maybe then it should contain both paths?

**Comment by sej7278** [comment]
you mean like this:

https://github.com/sudar/Arduino-Makefile/blob/master/arduino-mk-vars.md#arduino_dir

:smile:

**Comment by thomaskilian** [comment]
Exactly :-)

**Comment by sej7278** [comment]
its only an example, and personally I think Blink is better, there's more info in there, although not for osx i guess.

https://github.com/sudar/Arduino-Makefile/blob/master/examples/Blink/Makefile

if you want to modify your PR with some explanatory text about the two versions, i'm sure we can merge it, thanks.

**Comment by thomaskilian** [comment]
Ok. I have changed all places I could find. Will send you a new PR

**Comment by thomaskilian** [comment]
Sorry, I'm new to this mechanism. Looks like my updates are already forwarded?

**Comment by sej7278** [comment]
yes that's how it works, you don't need a new PR, it just munges together all your commits.

it looks like you've missed the opening triple-backticks ``` at line 148 as the "On Linux...." bit is still in the code tags.

could you also add something to HISTORY.md for example:

```
- Fix: Document OSX 1.0/1.6 ARDUINO_DIR differences (https://github.com/thomaskilian)
```

File: README.md [hunk 1]
```diff
@@ -144,6 +144,11 @@ On the Mac you might want to set:
     BOARD_TAG     = mega2560
 ```
 
+On the Mac with IDE 1.5+ it's like above but with
+
```
Comment (sej7278) [inline README.md]: needs backticks here ```

**Comment by thomaskilian** [comment]
Thanks, will do so.

**Pull request merged**